joss/paper.md

```diff
@@ -35,7 +35,9 @@ bibliography: papers.bib
 
-Implementations of the Dupuit-Forcheimer model often encounter numerical instabilities as the water table intersects the surface and groundwater return flow occurs through a seepage face, due to the presence of a discontinuity in the energy gradient from inside the hillslope (where it is determined by the water table) to the seepage face (where it is determined by the topography). @marcais_dynamic_2017 introduced a regularization that smooths the transition between subsurface flow and surface flow. Although introduced for numerical stability and not based on physical principles, this smoothing may reproduce the effect of subgrid heterogeneity where saturation within a grid cell is unlikely to be homogenous or well-reproduced by a binary condition (saturated vs unsaturated).
+Groundwater models of varying complexity are available for different purposes. Fully coupled groundwater and surface water models such as PARFLOW [@kollet_integrated_2006], CATHY [@camporese_surface-subsurface_2010], and PIHM [@qu_semidiscrete_2007] solve the three-dimensional Richards equation for variably saturated flow, and couple this with precipitation and runoff components. MODFLOW [@langevin_modflow_2019] also solves the three dimensional Richards equation, and may be coupled to precipitation and runoff models, as in GSFLOW [@regan_gsflow_2018]. More parsimonious models are also available that solve the hillslope storage Boussinesq equations, which model groundwater flow in quasi-two dimensional hillslopes [@marcais_dynamic_2017, @broda_low-dimensional_2012] using the Dupuit-Forcheimer approximation. Two-dimensional implementations of the Boussinesq model are common, but do not appear to be widely available as open source packages. The simplicity and computational efficiency of this method is advantageous for capturing the first-order effects of groundwater flow on other Earth surface processes. More sophisticated groundwater models may be necessary depending on the hydrological features that the user intends to capture. 
+
+Implementations of the Boussinesq model often encounter numerical instabilities as the water table intersects the surface and groundwater return flow occurs through a seepage face, due to the presence of a discontinuity in the energy gradient from inside the hillslope (where it is determined by the water table) to the seepage face (where it is determined by the topography). @marcais_dynamic_2017 introduced a regularization that smooths the transition between subsurface flow and surface flow. Although introduced for numerical stability and not based on physical principles, this smoothing may reproduce the effect of subgrid heterogeneity where saturation within a grid cell is unlikely to be homogenous or well-reproduced by a binary condition (saturated vs unsaturated).
```

Comment:
* Should "instabilities as the" be "instabilities where" or "instabilities when". I'm not entirely certain, so you should keep it as is if you feel strongly. 
* I would split the first sentence into two changing ", due" to ". This is due"

Reply:
Addressed in most recent commit!